### DISCUSSION TITLE
The Future of wappalyzergo: Adapting to Wappalyzer's Closed Source Status

### DISCUSSION THREAD

**Discussion by ceylanb** [projectdiscovery/wappalyzergo#56] [General]
As we know Wappalyzer is no longer open source [[Source]](https://news.ycombinator.com/item?id=37236746). I'm curious about the future plans of Wapplyzergo's maintainers and contributors. Do they anticipate the community surrounding Wappalyzergo will address this challenge, and what are their thoughts on this matter?

**Comment by ehsandeep** [comment]
@ceylanb, thanks for bringing this discussion! we are aware of this change and httpx will keep identifying technology based on the [data available](https://github.com/projectdiscovery/wappalyzergo) before it becomes private.

Unfortunately, the detection of new technology won't happen unless we switch to an alternative OSS project, not sure if there is any? that maintains & continues incorporating information about the latest technology.
